src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestCase.java

```diff
@@ -2168,6 +2168,122 @@ public void test_merge_strategy_correct_fail() throws Exception {
         }
     }
 
+    @Bug(12402)
```

Comment:
It not critical i think @MarkEWaite will change it before merge. Don't copy what you see without understanding.` IDE shows strikethrough text, then look at description that describes why it deprecated.